lexicons/app/bsky/actor/searchActors.json

```diff
@@ -4,16 +4,19 @@
   "defs": {
     "main": {
       "type": "query",
-      "description": "Find actors matching search criteria.",
+      "description": "Find actors (profiles) matching search criteria.",
       "parameters": {
         "type": "params",
         "properties": {
-          "term": { "type": "string" },
+          "term": {
```

Comment:
I would support phasing out `term` and phasing in `q`

I think "term" makes sense when it's simple search on a word & no additional syntax is allowed

`q` fits nicely when we allow for special query syntax (like Lucene syntax)

Reply:
cool, I updated all the instances of 'term' as a search query with 'q', and marked the old 'term' fields as "DEPRECATED" in description. This impacted an admin route as well. Idea is that for a short transition we'll fall back to "term" if "q" is empty, and maybe eventually nuke them (if/when we are feeling comfortable breaking query params in Lexicons in a small way, like a v1.0 release)